low-level/sspl_ll_d

```diff
@@ -713,6 +716,26 @@ def send_thread_controller_request(state):
 #     logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
 #     logger.setLevel(log_level)
 
+def initialize_iem():
+    try:
```

Comment:
Any pre-existing IEM-INIT FAILED file should be cleaned up here, before trying init at SSPL start

Reply:
done